src/services/userTransform.ts

```diff
@@ -87,6 +88,11 @@ export class UserTransformService {
           } as ProcessorTransformationResponse);
           return transformedEvents;
         }
+        stats.counter('user_transform_input_events', events.length, { workspaceId });
```

Comment:
Wouldn't this not allow us to see the issue if we log it at a workspace level ?

Reply:
Or maybe we don't want to know per transformation, generally what's the ratio is at the workspaceId level is something we are interested in ?